runtime/integration-tests/src/xcm/setup.rs

```diff
@@ -20,7 +20,9 @@ pub const ALICE: [u8; 32] = [4u8; 32];
 pub const BOB: [u8; 32] = [5u8; 32];
 
 /// Parachain Ids
+pub const PARA_ID_DEVELOPMENT: u32 = 2088;
```

Comment:
The most straightforward would probably be to just import the `crate::chain::centrifuge::altair::PARA_ID`

Reply:
Unfortunately that doesn't work since these tests currently can only run within the `runtime-development` flag. Once we add XCM to altair (next step), we can make that switch. wdyt?